sources/packages/web/src/components/aest/StudentApplicationOfferingChangeDetails.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-card>
+    <v-container>
+      <body-header :title="studentName" class="m-1" />
+      <header-title-value
+        class="mb-2"
+        title="Application#"
+        :value="applicationNumber"
+      ></header-title-value>
+      <header-title-value
+        class="mb-6"
+        title="Location"
+        :value="locationName"
+      ></header-title-value>
+      <content-group-info
+        ><div class="mb-2">
```

Comment:
`content-group-info` is already a `div` with `my-3`, do we need this `div` and this margin class?